Source/CameraView/CameraView.swift

```diff
@@ -9,7 +9,7 @@ protocol CameraViewDelegate: class {
 }
 
 class CameraView: UIViewController, CLLocationManagerDelegate {
-
+  
```

Comment:
@aashishdhawan What is going on here? 🤔
We have an Xcode plugin installed to remove such white spaces on save, you might want to try it - https://github.com/zenangst/MarvinXcode